gr-blocks/lib/rotator_cc_impl.cc

```diff
@@ -40,6 +48,33 @@ void rotator_cc_impl::set_phase_inc(double phase_inc)
     d_r.set_phase_incr(exp(gr_complex(0, phase_inc)));
 }
 
+void rotator_cc_impl::handle_phase_inc_msg(pmt::pmt_t msg)
+{
+    const pmt::pmt_t inc_key = pmt::intern("inc");
```

Comment:
these could be globally defined

Reply:
for example `static const`, not just `const`

Reply:
Done :+1: